http/README.md

```diff
@@ -0,0 +1,138 @@
+# @motorcycle/http
+
+> Motorcycle Components and functions for dealing with HTTP
+
+This is a thin abstraction and helpful functions built on top of
+[`most-request`](https://github.com/TylorS/most-request) which uses
+[`superagent`](https://github.com/visionmedia/superagent) under the hood.
+
+## Let me have it!
+```sh
+yarn add @motorcycle/http
+# or
+npm install --save @motorcycle/http
+```
+
+## API
+
+- All functions of arity 2 or more are curried for you!
+- All types in the API section not described can be found in the [Types](#types) section.
```

Comment:
Types not described in the API section can be found in the [Types](#types) section.